arches/app/media/js/models/card-widget.js

```diff
@@ -91,7 +91,7 @@ export default AbstractModel.extend({
                 var configJSON = {};
                 var config = this.get('config');
                 _.each(this.configKeys(), function(key) {
-                    configJSON[key] = koMapping.toJS(config[key]);
+                    configJSON[key] = ko.unwrap(config[key]);
```

Comment:
I notice that we had made this change in reverse some years ago: https://github.com/archesproject/arches/commit/27bc9ef3491bd365f0f864a8c7543a1f2e8c58bc
I think that this was specifically to support configuration objects with complex objects (I think the map widget is an example of this).  I'm not able to test this at the moment as the designer page is broken for me on this branch, but I'm concerned about the impact of this change on widgets with complex configuration objects.

Reply:
oooo right you are. Made a more subtle change that should jive